mantidproject/mantid
# Fix tooltip colours on RHEL7

Fixes the colour pallete so that the hints are readable on RHEL7.

> On Linux (tested on Fedora 26-7 and Ubuntu 14.4) the palette colour for
> ToolTipBase has no effect on the colour of tooltips, but does change
> the colour of 'What's This' boxes and and Line Edit hints. The palette
> colour for ToolTipText on the other hand affects all three of these.
>
> The default pallete shows light text on a pale background which, although not
> affecting tooltips, makes LineEdit hints and 'What's This' boxes difficuilt
> if not impossible to read.
> 
> Changing the tooltip text to a darker colour fixes the problem for 'LineEdit'
> hints and 'What's This' boxes but creates one for ordinary tooltips.
>
> Setting the background colour to a darker colour works fine on
> Fedora 26-7+ and Ubuntu 14.04 but not for RHEL7 where the tooltip text
> colour is also dark.
>
>  At the moment the fix is to simply hardcode the values to a dark on
>  light colour scheme which works consistently on Fedora 26-7 and Ubuntu
>  14.04 where ToolTipBase has no effect

**To test:**
Check that the tool-tips, line edit hints and whats this help have appropriate colours.



Fixes #21921 . 

**Release Notes** 


---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
